ansible_collections/arista/avd/roles/eos_designs/python_modules/core_interfaces_and_l3_edge/utils.py

```diff
@@ -161,15 +161,22 @@ def _get_p2p_data(self, p2p_link: dict) -> dict:
         node_child_interfaces = convert_dicts(node_child_interfaces, primary_key="node", secondary_key="interfaces")
         if member_interfaces := get_item(node_child_interfaces, "node", self.shared_utils.hostname, default={}).get("interfaces"):
             # Port-channel
-            peer_member_interfaces = get_item(
+            pc_id = get_item(node_child_interfaces, "node", self.shared_utils.hostname, default={}).get("channel_id")
```

Comment:
Instead of using get_item multiple times, you should refactor to

```py
if (node_data := get_item(node_child_interfaces, "node", self.shared_utils.hostname)) is not None and (member_interfaces := node_data.get("interfaces")):
```
Then you have node_data available to get the channel_id. Please don't use abbreviations like pc_id.

Reply:
- `pc_id` was already in use before this PR. I have renamed it to `portchannel_id`.
- I have re-written this to simply the implementation, please let me know what you think.